corehq/apps/api/domain_metadata.py

```diff
@@ -50,6 +51,10 @@ class DomainMetadataResource(HqBaseResource):
     calculated_properties = fields.DictField()
     domain_properties = fields.DictField()
 
+    # using the default resource dispatch function to bypass our authorization for internal use
+    def dispatch(self, request_type, request, **kwargs):
+        return Resource.dispatch(self, request_type, request, **kwargs)
```

Comment:
yeah is this not a security hole or how does it work exactly? I would have thought it should `401` or something, but maybe that happens in the super call?